common/metrics/tags.go

```diff
@@ -290,3 +293,8 @@ type ReasonString string
 func ReasonTag(value ReasonString) Tag {
 	return &tagImpl{key: reason, value: string(value)}
 }
+
+// ReplicationTaskTypeTag returns a new replication task type tag.
+func ReplicationTaskTypeTag(value serverenumbspb.ReplicationTaskType) Tag {
```

Comment:
do we really need a tag here? we try to be pretty sparing with metric cardinality. (well, we haven't tried, but we are going forwards)

Reply:
Cardinality is limited by proto enum definitions and this is useful info IMHO.